workflows/Purge_Office_365_Emails_with_Slack/workflow.spec.yaml

```
@@ -0,0 +1,23 @@
+extension: workflow
+products: ["insightconnect"]
+name: Purge_Office_365_Emails_with_Slack
+title: "Purge Office 365 Emails with Slack"
+description: "This workflow will purge all emails in an organization matching given search criteria by sending a notification to InisghtConnect with Slack. The workflow will look for any emails that match the body, subject, or from contents given. \n\nIt can either automatically delete these emails or prompt the user for further action."
```

Short description here
```suggestion
description: "This workflow will purge all emails in an organization matching given search criteria by sending a notification to InisghtConnect with Slack."
```